From e7923413a41e022434265e6f2a26078c50abeb68 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Wed, 21 Mar 2012 14:29:19 +0800
Subject: [PATCH 287/442] ENGR00177387 Fix build error of missing videodev.h

videodev.h is removed from latest kernel header file.
Replace videodev.h as videodev2.h to fix build error.

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 test/mxc_mpeg4_enc_test/cam2mpeg4_cif.c | 2 +-
 test/mxc_mpeg4_enc_test/cam2mpeg4_vga.c | 2 +-
 test/mxc_v4l2_test/csi_v4l2_capture.c   | 4 ++--
 test/mxc_v4l2_test/csi_v4l2_overlay.c   | 4 ++--
 test/mxc_v4l2_test/mxc_v4l2_capture.c   | 4 ++--
 test/mxc_v4l2_test/mxc_v4l2_output.c    | 4 ++--
 test/mxc_v4l2_test/mxc_v4l2_overlay.c   | 4 ++--
 test/mxc_v4l2_test/mxc_v4l2_still.c     | 4 ++--
 test/mxc_v4l2_test/mxc_v4l2_tvin.c      | 1 -
 test/mxc_vpu_test/capture.c             | 4 ++--
 test/mxc_vpu_test/vpu_test.h            | 2 +-
 11 files changed, 17 insertions(+), 18 deletions(-)

diff --git a/test/mxc_mpeg4_enc_test/cam2mpeg4_cif.c b/test/mxc_mpeg4_enc_test/cam2mpeg4_cif.c
index c87182b..74a397e 100644
--- a/test/mxc_mpeg4_enc_test/cam2mpeg4_cif.c
+++ b/test/mxc_mpeg4_enc_test/cam2mpeg4_cif.c
@@ -17,7 +17,7 @@
 #include <fcntl.h>
 
 #include <errno.h>
-#include <linux/videodev.h>
+#include <linux/videodev2.h>
 #include <string.h>
 #include <malloc.h>
 
diff --git a/test/mxc_mpeg4_enc_test/cam2mpeg4_vga.c b/test/mxc_mpeg4_enc_test/cam2mpeg4_vga.c
index 5d9f7b2..96facf6 100644
--- a/test/mxc_mpeg4_enc_test/cam2mpeg4_vga.c
+++ b/test/mxc_mpeg4_enc_test/cam2mpeg4_vga.c
@@ -14,7 +14,7 @@
 /* For linear memory allocation */
 #include "memalloc.h"
 
-#include <linux/videodev.h>
+#include <linux/videodev2.h>
 #define TEST_BUFFER_NUM 3
 
 struct cmd_line
diff --git a/test/mxc_v4l2_test/csi_v4l2_capture.c b/test/mxc_v4l2_test/csi_v4l2_capture.c
index fedee1e..d92db04 100644
--- a/test/mxc_v4l2_test/csi_v4l2_capture.c
+++ b/test/mxc_v4l2_test/csi_v4l2_capture.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2009 Freescale Semiconductor, Inc. All rights reserved.
+ * Copyright 2009-2012 Freescale Semiconductor, Inc. All rights reserved.
  */
 
 /*
@@ -39,7 +39,7 @@ extern "C"{
 #include <stdlib.h>
 #include <asm/types.h>
 #include <linux/fb.h>
-#include <linux/videodev.h>
+#include <linux/videodev2.h>
 #include <sys/mman.h>
 #include <sys/time.h>
 #include <string.h>
diff --git a/test/mxc_v4l2_test/csi_v4l2_overlay.c b/test/mxc_v4l2_test/csi_v4l2_overlay.c
index f30590c..5edae92 100644
--- a/test/mxc_v4l2_test/csi_v4l2_overlay.c
+++ b/test/mxc_v4l2_test/csi_v4l2_overlay.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2009 Freescale Semiconductor, Inc. All rights reserved.
+ * Copyright 2009-2012 Freescale Semiconductor, Inc. All rights reserved.
  */
 
 /*
@@ -39,7 +39,7 @@ extern "C"{
 #include <stdio.h>
 #include <stdlib.h>
 #include <asm/types.h>
-#include <linux/videodev.h>
+#include <linux/videodev2.h>
 #include <linux/fb.h>
 #include <sys/mman.h>
 #include <math.h>
diff --git a/test/mxc_v4l2_test/mxc_v4l2_capture.c b/test/mxc_v4l2_test/mxc_v4l2_capture.c
index 50cdecc..fc18220 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_capture.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_capture.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2011 Freescale Semiconductor, Inc. All rights reserved.
+ * Copyright 2004-2012 Freescale Semiconductor, Inc. All rights reserved.
  */
 
 /*
@@ -38,7 +38,7 @@ extern "C"{
 #include <stdio.h>
 #include <stdlib.h>
 #include <asm/types.h>
-#include <linux/videodev.h>
+#include <linux/videodev2.h>
 #include <sys/mman.h>
 #include <string.h>
 #include <malloc.h>
diff --git a/test/mxc_v4l2_test/mxc_v4l2_output.c b/test/mxc_v4l2_test/mxc_v4l2_output.c
index 8c3517e..4884d63 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_output.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_output.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2011 Freescale Semiconductor, Inc. All rights reserved.
+ * Copyright 2004-2012 Freescale Semiconductor, Inc. All rights reserved.
  */
 
 /*
@@ -35,7 +35,7 @@ extern "C"{
 #include <fcntl.h>
 #include <sys/ioctl.h>
 #include <unistd.h>
-#include <linux/videodev.h>
+#include <linux/videodev2.h>
 #include <sys/mman.h>
 #include <math.h>
 #include <string.h>
diff --git a/test/mxc_v4l2_test/mxc_v4l2_overlay.c b/test/mxc_v4l2_test/mxc_v4l2_overlay.c
index 1d583a9..85fd546 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_overlay.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_overlay.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2011 Freescale Semiconductor, Inc. All rights reserved.
+ * Copyright 2004-2012 Freescale Semiconductor, Inc. All rights reserved.
  */
 
 /*
@@ -39,7 +39,7 @@ extern "C"{
 #include <stdio.h>
 #include <stdlib.h>
 #include <asm/types.h>
-#include <linux/videodev.h>
+#include <linux/videodev2.h>
 #include <linux/fb.h>
 #include <sys/mman.h>
 #include <math.h>
diff --git a/test/mxc_v4l2_test/mxc_v4l2_still.c b/test/mxc_v4l2_test/mxc_v4l2_still.c
index 662b4b7..78fb357 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_still.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_still.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2010 Freescale Semiconductor, Inc. All rights reserved.
+ * Copyright 2004-2012 Freescale Semiconductor, Inc. All rights reserved.
  */
 
 /*
@@ -35,7 +35,7 @@ extern "C"{
 #include <stdio.h>
 #include <stdlib.h>
 #include <asm/types.h>
-#include <linux/videodev.h>
+#include <linux/videodev2.h>
 #include <sys/mman.h>
 #include <string.h>
 #include <malloc.h>
diff --git a/test/mxc_v4l2_test/mxc_v4l2_tvin.c b/test/mxc_v4l2_test/mxc_v4l2_tvin.c
index 9a42951..d47b4ab 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_tvin.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_tvin.c
@@ -37,7 +37,6 @@ extern "C"{
 #include <sys/time.h>
 #include <unistd.h>
 #include <asm/types.h>
-#include <linux/videodev.h>
 #include <linux/videodev2.h>
 #include <sys/mman.h>
 #include <math.h>
diff --git a/test/mxc_vpu_test/capture.c b/test/mxc_vpu_test/capture.c
index 713b46d..39a24fe 100644
--- a/test/mxc_vpu_test/capture.c
+++ b/test/mxc_vpu_test/capture.c
@@ -1,6 +1,6 @@
 
 /*
- * Copyright 2004-2011 Freescale Semiconductor, Inc.
+ * Copyright 2004-2012 Freescale Semiconductor, Inc.
  *
  * Copyright (c) 2006, Chips & Media.  All rights reserved.
  */
@@ -21,7 +21,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <linux/videodev.h>
+#include <linux/videodev2.h>
 #include "vpu_test.h"
 
 #define TEST_BUFFER_NUM 3
diff --git a/test/mxc_vpu_test/vpu_test.h b/test/mxc_vpu_test/vpu_test.h
index d1da7ac..cf5bdea 100644
--- a/test/mxc_vpu_test/vpu_test.h
+++ b/test/mxc_vpu_test/vpu_test.h
@@ -16,7 +16,7 @@
 #ifndef _DEC_H
 #define _DEC_H
 
-#include <linux/videodev.h>
+#include <linux/videodev2.h>
 #include <pthread.h>
 #include <errno.h>
 #include <stdint.h>
-- 
1.8.0

